src/components/FilterSizes.js

```diff
@@ -0,0 +1,114 @@
+                    <div className="popup filler" id="filler-one"></div>
+                    <div className="popup" id="four" onMouseEnter={this.hover} onMouseLeave={this.leave}></div>
+                    <div className="popup filler" id="filler-two"></div>
+                    <div className="popup" id="five" onMouseEnter={this.hover} onMouseLeave={this.leave}></div>
+                    <img src={size} alt="Filters and layers of a CNN" id="size-img"/>
+                    <div className="popup-text-container" id="input">
+                        <p>
+                            Notice how our input, a 6 x 6 RGB image, is split into three 6 x 6 layers.
+                        </p>
+                    </div>
+                    <div className="popup-text-container" id="convolve">
+                        <p>This is the symbol for convolution. The filters <em>convolve</em> over the image.</p>
+                    </div>
+                    <div className="popup-text-container" id="filters">
+                        <p>
+                            Filter 1 and Filter 2 both have three layers. The first layer of each filter
+                            convolves over the first layer of the input image, the second layer of each filter convolves over the
+                            second layer of the input image, and so on. Then, for each filter, the three resulting matrices are summed up to produce an output of one layer.
+                        </p>
+                        <p>In order for our outputs to be the same size, Filter 1 and Filter 2 must be the same size as each other.</p>
+                    </div>
+                    <div className="popup-text-container" id="output-separate">
+                        <p>
+                            Our 3 x 3 filters can only move along each row of the input image four times until it hits the right side (try working it out yourself!).
+                            Similarly, it can only move down a row four times until it hits the bottom, giving our output (also known as a <em>feature map</em>) a size of 4 x 4.
+
+                        </p>
+                        <p>If we use filters of bigger sizes, our output matrix will be smaller, and will capture less information.
+                            As a result, we should be careful when choosing our filter size.
+                        </p>
+                    </div>
+                    <div className="popup-text-container" id="output-together">
+                        <p>
+                            Each filter produced an output of size 4 x 4 x 1. We then stack the result of each filter convolution, to produce an output of size 4 x 4 x 2.
+
+                        </p>
+                        <p>
+                            This output now acts as the input to the next layer of our CNN!
+                        </p>
+                    </div>
```

Comment:
To avoid code duplication here, you may find it easier to generate this with `.map` and a constant array of objects, but no pressure if you think that's confusing.